src/components/order-service/order-service.controller.ts

```diff
@@ -71,6 +71,7 @@ export class OrderServiceController implements angular.IController {
 
     this.ctrl.appToBind = null;
     this.ctrl.configStepValid = true;
+    this.ctrl.lessThanTwoServicePlans = _.size(this.ctrl.servicePlans) < 2;
```

Comment:
Maybe call it `hasMultiplePlans` and flip the logic? I feel like that reads better.